src/java.base/share/classes/java/time/format/SignStyle.java

```diff
@@ -123,17 +123,13 @@
      * @return
      */
     boolean parse(boolean positive, boolean strict, boolean fixedWidth) {
-        switch (ordinal()) {
-            case 0: // NORMAL
-                // valid if negative or (positive and lenient)
-                return !positive || !strict;
-            case 1: // ALWAYS
-            case 4: // EXCEEDS_PAD
-                return true;
-            default:
-                // valid if lenient and not fixed width
-                return !strict && !fixedWidth;
-        }
+        return switch (ordinal()) {
+            // valid if negative or (positive and lenient)
```

Comment:
The comment should apply only to the `case 0`.

Reply:
Comment corrected. Please see commit 2ae4a57